examples/public-vpc/main.tf

```diff
@@ -0,0 +1,129 @@
+resource "aws_vpc" "test" {
```

Comment:
I want to make sure no one is going direct to prod with our examples so naming everything test is part of how I'm achieving it. We're also sourcing the module from the repo rather than terraform's module registry (which is best practice for examples that are dual purposed to CI) so that should hopefully put another barrier in the way of copypasta

Reply:
fair enough.